### ISSUE TITLE
quotes are replaced in the query pane - causes syntax errors

### ISSUE THREAD

**Issue by aokolish** [woollyapp/woolly#5]
When you type a query into the query pane, quotes get replaced by curly quotes - it causes syntax errors. When I paste in a query, the quotes are not replaced.

![pasted_image_11_13_13__3_40_pm-10](https://f.cloud.github.com/assets/705514/1537047/75e6df3e-4cbd-11e3-8c6a-35fc61936920.jpg)

issue:

```
ERROR:  syntax error at or near "id”"
LINE 1: select id as “user id” from users;
```

**Comment by kevintom** [comment]
found this, 

http://stackoverflow.com/questions/19801601/nstextview-with-smart-quotes-disabled-still-replaces-quotes

i guess they changed the default behavior in Mavericks and I didn't notice.

**Comment by aokolish** [comment]
Good find. I guess I can fix that for myself:

> Also in Mavericks there is smart quotes enabled by default in System Preferences → Keyboard → Text. You can disable this as a personal preference.

**Comment by kevintom** [comment]
should  be fixed in 1.0.3
